config/webpack/common.webpack.config.js

```diff
@@ -75,8 +75,9 @@ function getWebpackConfig(skyPagesConfig, argv = {}) {
       rules: [
         {
           enforce: 'pre',
-          test: /runtime\/config\.ts$/,
-          loader: outPath('loader', 'sky-app-config')
+          test: /config\.ts$/,
+          loader: outPath('loader', 'sky-app-config'),
+          include: outPath('runtime')
```

Comment:
Just curious about the different syntax here.

Reply:
it will ONLY look in the root's runtime folder, instead of "any" runtime folder.

Reply:
Not a huge time saver, but recommended by Webpack docs to be more specific with loaders. https://webpack.js.org/guides/build-performance/#loaders